client/buildpacks.go

```diff
@@ -161,6 +155,23 @@ func (b *buildpacksFunctionServer) run() (func(), error) {
 	}, nil
 }
 
+func (b *buildpacksFunctionServer) getRuntimeVars() []string {
+	var variables = strings.Split(b.envs, ",")
+	runtimeVars := []string {"run",
+	"--network=host",
+	// TODO: figure out why these aren't getting set in the buildpack.
+	"--env=FUNCTION_TARGET=" + b.target,
+	"--env=FUNCTION_SIGNATURE_TYPE=" + b.funcType,}
+
+	for _,s := range variables{
+		if s != "" {
+			runtimeVars = append(runtimeVars, fmt.Sprintf("--env=%s",s))
+		}
+	}
+	
+	return append(runtimeVars, image)
```

Comment:
This parameter feels like it should go with the calling function? If "docker run image" can be the first 3 words of the command?

Reply:
image has to be the last parameter for docker run command. I will just rename this function to getDockerRunCommand? wdyt?